modules/govuk_jenkins/files/var/lib/jenkins/groovy_scripts/govuk_jenkinslib.groovy

```diff
@@ -34,11 +46,6 @@ def checkoutFromGitHubWithSSH(String repository, String org = 'alphagov', String
  * branch.
  */
 def mergeMasterBranch() {
-  def isCurrentCommitOnMaster = sh(
-    script: 'git rev-list origin/master | grep $(git rev-parse HEAD)',
-    returnStatus: true
-  ) == 0
-
   if (isCurrentCommitOnMaster) {
```

Comment:
I think you need the parentheses in zero-arg function calls: http://groovy-lang.org/semantics.html#_optional_parentheses

Ditto for line 137.